sdk/storage/storage-blob/test/perfstress/track-2/storageTest.spec.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+
+import { BlobServiceClient, ContainerClient } from "../../../src";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export abstract class StorageBlobTest<TOptions> extends PerfStressTest<TOptions> {
+  blobServiceClient: BlobServiceClient;
+  containerClient: ContainerClient;
+  static containerName = `newcontainer${new Date().getTime()}`;
+
+  constructor() {
+    super();
+    this.blobServiceClient = BlobServiceClient.fromConnectionString(
+      StorageBlobTest.getEnvVar("STORAGE_CONNECTION_STRING")
+    );
+    this.containerClient = this.blobServiceClient.getContainerClient(StorageBlobTest.containerName);
+  }
+
+  public async globalSetup() {
+    await this.containerClient.create();
```

Comment:
Call super setup and cleanup

Reply:
As said before.. This is the base class that extends PerfStressTest, in which globalSetup and the other methods aren't implemented.
Hence, not required.